src/inc/regdisp.h

```diff
@@ -11,7 +11,7 @@ class Thread;
 #endif // DEBUG_REGDISPLAY
 
 
-#if defined(_TARGET_X86_)
+#if defined(_TARGET_X86_) && !defined(WIN64EXCEPTIONS)
```

Comment:
By mistake, I thought that ``GetRegdisplayReturnValue`` is available only for ``WIN64EXCEPTIONS``.